src/ReleaseTests/NoUnusedTemporaryVariablesLeftTest.class.st

```diff
@@ -23,8 +23,9 @@ NoUnusedTemporaryVariablesLeftTest >> tearDown [
 NoUnusedTemporaryVariablesLeftTest >> testNoUnusedTemporaryVariablesLeft [
 	"Fail if there are methods who have unused temporary variables"
 	| found validExceptions remaining |
-	found := (SystemNavigation default allBehaviors flatCollect: #localMethods)
-		select: [ :m | m ast temporaries anySatisfy: [ :x | x binding isUnused ] ].
+	found := SystemNavigation default allMethodsSelect: [ :m | 
+		((m numTemps - m numArgs) > 0) and: [  
```

Comment:
Maybe we can add #numTempsExceptArgs or with some better name

Reply:
yes, I was thinking about it.. it is really bad that the compiledMethod and the AST have a different model of what a temp is...